Gemfile

```diff
@@ -29,18 +26,29 @@ gem "bootsnap", require: false
 # Use Rack CORS for handling Cross-Origin Resource Sharing (CORS), making cross-origin Ajax possible
 # gem "rack-cors"
 
+# Discord Bot
 gem "discordrb", "~> 3.5.0"
 
+# LLMs
+gem "ruby-openai", "~> 6.3.0"
+gem "langchainrb"
+
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri windows ], require: "debug/prelude"
```

Comment:
I'm interested in why this was changed?

Reply:
Same goes here, windows isn't a valid platform everywhere.